src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.ViewButton.cshtml

```diff
@@ -0,0 +1,12 @@
+@using OrchardCore.ContentManagement
+@inject IContentManager ContentManager
+
+@{
+    ContentItem contentItem = Model.ContentItem;
+    var hasPublished = await ContentManager.HasPublishedVersionAsync(contentItem);
+}
+
+@if (hasPublished)
```

Comment:
External url in blank page, in front of the autoroute field

Reply:
What is the best way to have the content item url from the AutoroutePart?

In the action, I could use this:
    <a display-for="@contentItem" class="btn btn-light" target="_blank" itemprop="">@T["View"]</a>